docs/reference/ilm/apis/explain.asciidoc

```diff
@@ -147,10 +150,11 @@ phase completes.
   "indices": {
     "test-000069": {
       "index": "test-000069",
+      "index_creation_date_millis": 1538475653281,
+      "index_age": "25.14s",
       "managed": true,
       "policy": "my_lifecycle3",
       "lifecycle_date_millis": 1538475653281,
-      "lifecycle_date": "2018-10-15T13:45:21.981Z",
```

Comment:
Are we not exposing the `lifecycle_date` anymore? That'd be a breaking change. I think we should keep exposing that.

Reply:
I didn't see it being exposed in the code so I thought I was updating the documentation to match what we currently do. What do we do in these cases?

Reply:
We are exposing it https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponse.java#L422

Reply:
Copy that, I checked the code further to see how it works and indeed there are paths that both are shown. I will put it back.